frontend/src/screens/Home.tsx

```diff
@@ -15,12 +15,70 @@ const Home = ({ navigation }): JSX.Element => {
   const audioSoundRef = React.useRef(new Audio.Sound());
   const [soundStatus, setSoundStatus] = useState<AVPlaybackStatusSuccess>()
   const [isPlaying, setIsPlaying] = useState<boolean>(false)
+  const [tracksMeta, setTracksMeta] = useState<any>()
+  const [refreshing, setRefreshing] = React.useState(false);
 
+  const onEndReached = React.useCallback(() => {
+    setRefreshing(true);
+
+    getRandomTracks(tracksMeta.next).then((data) => {
+      
+      console.log(data)
+      
+      setTracks([...tracks, ...trackItemMapper(data.tracks.items)])
+      setTracksMeta({
+        'limit': data.tracks.limit,
+        'offset': data.tracks.offset,
+        'previous': data.tracks.previous,
+        'next': data.tracks.next,
+        'total': data.tracks.total
+      })
+    },
+    (err) => {
+      console.log(err)
+    })
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 2000);
+
+  }, [tracksMeta])
+
+  const onRefresh = React.useCallback(() => {
+    setRefreshing(true);
+    getRandomTracks().then((data) => {
+      
+      console.log(data)
+      
+      setTracks(trackItemMapper(data.tracks.items))
+      setTracksMeta({
+        'limit': data.tracks.limit,
+        'offset': data.tracks.offset,
+        'previous': data.tracks.previous,
+        'next': data.tracks.next,
+        'total': data.tracks.total
+      })
+    },
+    (err) => {
+      console.log(err)
+    })
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 2000);
+  }, []);
 
   useEffect(() => {
-    getPlaylist("1LwKg8pkx71G83WgOfvlLZ").then((data) => {
+    getRandomTracks().then((data) => {
+      
       console.log(data)
```

Comment:
I notice that you have included many logs in your code. Would it be possible to remove them, as they are only for debugging purposes?